src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from 'react';
+import Map from '../../Map/Map.js';
+import { AccountType } from '../../Enums';
+import { StringFormat } from '../../Enums';
+
+class OrganizationDetails extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isEditing: false
+        };
+    }
+
+    render() {
+        return (
+            <div className="scs-0">
+                <div className="scs-0-image">
+                    <Map
+                        marginLeft="20px" 
+                        height="150px" 
+                        width="70%" 
+                        marginTop="10px" 
+                        marginRight="0px"
+                        marginBottom="0px"
+                        address={this.props.account.address}
+                    />
+                </div>
+
+                {!this.state.isEditing ?
+
+                    <div className="scs-0-content">
+                        <h5>Organization Details</h5>
+                        <h6>{this.props.account.name}</h6>
+                        <div className="scs-spacing" />
+                        <h6>Address: {this.props.account.address.street1}, {this.props.account.address.street2 ? <span>{this.props.account.address.street2},</span> : <span />} {this.props.account.address.city}, {this.props.account.address.state} - {this.props.account.address.zipcode}</h6>
+                        {this.props.account.phone ? <h6>Phone: {this.props.account.organizationPhone}</h6> : <span />}
```

Comment:
(did I add phone here in the sketch? lol) there are no phones for organization, so can remove